### PR TITLE
Bug/foaf fix

### PR DESCRIPTION
**Pull request by FordMcLeod**
Resolves visibility issues

### CODE REVIEW

File: posts/helpers.py [hunk 1]
```diff
@@ -32,9 +32,10 @@ def parse_id_from_url(url):
     Parses a user id from user urls in the form:
     https://example.com/author/f3be7f78-d878-46c5-8513-e9ef346a759d/
     """
-    user_url = url.split('/author/')[-1]
-    user_url = user_url[:-1] if user_url[-1] == '/' else user_url
-    return user_url
+    parsed = urlparse(url)
+    path = parsed.path.strip('/')
```
Comment (MichaelParadis) [inline posts/helpers.py:36]: Is this intentional? Don't want us to revert to using something that had issues before

File: posts/models.py [hunk 2]
```diff
@@ -180,7 +180,7 @@ def get_ext_feed(self, requestor):
         if self.trailing_slash:
             url = self.api + '/author/posts/'
         else:
-            url = self.api + '/author//posts'
```
Comment (MichaelParadis) [inline posts/models.py:183]: I am upset that I am pretty sure I wrote this line

**Review by MichaelParadis — APPROVED** [review]
Is the zip file needed? If so all good to go

File: posts/helpers.py (hunk 1 above)
Comment (FordMcLeod) [reply posts/helpers.py:36]: Yes, yours and kieters fix breaks on viewing external users. When I reverted to the previous implementation it worked

File: posts/models.py (hunk 2 above)
Comment (FordMcLeod) [reply posts/models.py:183]: lmao it happens

File: posts/helpers.py (hunk 1 above)
Comment (FordMcLeod) [reply posts/helpers.py:36]: Mike it works  FINE

Comment (FordMcLeod) [reply posts/helpers.py:36]: <3

**Pull request merged**